modules/seqexec/server/src/main/scala/seqexec/server/gmos/GmosKeywordReader.scala

```diff
@@ -131,60 +130,56 @@ object GmosKeywordReaderDummy {
 }
 
 object GmosKeywordReaderEpics {
-  def apply[F[_]: Sync: LiftIO]: GmosKeywordReader[F] = new GmosKeywordReader[F] {
-    private val F = implicitly[Sync[F]]
-    private val sys = GmosEpics.instance
+  def apply[F[_]: Sync](sys: GmosEpics[F]): GmosKeywordReader[F] = new GmosKeywordReader[F] {
```

Comment:
I tend to give these things names like `fromGmosEpics` rather than using `apply`. Not a big deal.